### PR TITLE
Create hook-shelve.py

### PR DESCRIPTION
**Pull request by Penaz91**
Added hook for the Shelve module.

### CODE REVIEW

File: PyInstaller/hooks/hook-shelve.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Hidden Imports for Shelve Module
```
Comment (matysek) [inline PyInstaller/hooks/hook-shelve.py]: @Penaz91 Thans for your hook. Could you please add here the license header similar to other hooks and a comment about the shelve version and OS you tested it with?

**Comment by Penaz91** [comment]
Added the copyright lines.
I tested this hook on Windows 7 x64 with Python 3.5.

**Comment by matysek** [comment]
> Windows 7 x64 with Python 3.5.

Please mention this directly in the hook file together with shelve version. So other users know what platforms it was/wasn't tested on.

**Comment by Penaz91** [comment]
I added a note on the OS and Python version used inside the file, but i couldn't find the version of Shelve, as it has no __version__ nor "pip freeze" reports its presence. Shelve is part of the standard Python Library.

**Comment by matysek** [comment]
> Shelve is part of the standard Python Library.

I see. Thanks.

**Pull request merged**